app/scripts/views/complete_sign_up.js

```diff
@@ -44,6 +45,10 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
       // cache the email in case we need to attempt to resend the
       // verification link
       this._email = this._account.get('email');
+      this._able = options.able;
+
+      this._experiments = new ExperimentInterface();
+      this._experiments.chooseExperiments();
```

Comment:
@zaach if we move it to initialize then can views that use the mixin just assume that `this._experiments ` is defined?